Module/dlg/store002.dlg.json

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
All of these shop dialogs appear to be identical. You can get away with a single dialog and use the snippets mentioned previously to cut down on all these redundant files.